src/fstar/FStar.Universal.fs

```diff
@@ -397,8 +397,7 @@ let tc_one_file
             maybe_restore_opts ();
             //AR: encode smt module and do post processing
             if (not (Options.lax())) then begin
-              FStar.SMTEncoding.Encode.encode_modul_from_cache env tcmod smt_decls;
-              post_smt_encoding ()
```

Comment:
Looks like `post_smt_encoding` can be deleted now.

Reply:
removed